client/Combatant/Combatant.ts

```diff
@@ -23,7 +23,6 @@ export interface Combatant {
     InitiativeGroup: KnockoutObservable<string>;
     Hidden: KnockoutObservable<boolean>;
     StatBlock: KnockoutObservable<StatBlock>;
-    GetInitiativeRoll: () => number;
```

Comment:
Why was this removed?

Reply:
JSLint complained that it was defined twice after I made the changes to the function definition on line 171. I'll see if that changes after making the changes you requested. If not, do you want me to ignore the JSLint warning and leave it there?